apps/researcher/src/app/[locale]/page.tsx

```diff
@@ -32,17 +33,39 @@ import {ElementType} from 'react';
 // Revalidate the page every n seconds
 export const revalidate = 60;
 
-interface FacetProps {
+interface FilterSetting {
   name: keyof SearchResult['filters'];
   searchParamType: SearchParamType;
+}
+
+interface FacetComponent {
```

Comment:
The interface `FacetComponent` also has a prop called `Component` - this makes the name of the interface somewhat harder to understand for me. Would just the name `Facet` work (i.e. a facet consists of a name and a component)?